app/controllers/active_matches_controller.rb

```diff
@@ -18,9 +18,17 @@ def index
     @matches = match_scope
     @current_step = params[:current_step]
     @current_program = params[:current_program]
+    @current_contact_type = params[:current_contact_type]&.to_sym
+    if current_user.can_view_all_matches?
+      @current_filter_contact = params[:current_filter_contact]
+      @current_filter_contact = @current_filter_contact.to_i if @current_filter_contact.present?
```

Comment:
I really wanted `params[:current_filter_contact]&.to_i` to return `nil` if the param was missing (or an empty string, I guess) but it was returning `0`. Ugly solution :\